src/config/config.js

```diff
@@ -14,8 +14,9 @@ const spotify = {
 };
 const azure = {
   // apiUrl: 'https://rogue-sound-poc-functions.azurewebsites.net/api',
-  // apiUrl: 'https://rogue-sound-func.azurewebsites.net/api',
-  apiUrl: 'https://roguesound.azurewebsites.net/api',
+  apiUrl: 'https://rogue-sound-func.azurewebsites.net/api',
```

Comment:
I'm not quite sure if this should be changed.

cc/ @jmolla31

Reply:
It's ok fttb, but we should move that into an environment variable or the .env file.

Reply:
Agree, for now I'd keep our usual URL and make a PR to be selected through the .env file

Reply:
Leaving for the next PR then.